github/git_blobs.go

```diff
@@ -38,6 +39,23 @@ func (s *GitService) GetBlob(ctx context.Context, owner string, repo string, sha
 	return blob, resp, err
 }
 
+// GetBlobRaw fetches a blob's contents from a repo.
+// Unlike GetBlob, it returns the raw bytes rather than the base64-encoded data.
+func (s *GitService) GetBlobRaw(ctx context.Context, owner, repo, sha1 string) ([]byte, error) {
+	u := fmt.Sprintf("repos/%v/%v/git/blobs/%v", owner, repo, sha1)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Accept", "application/vnd.github.v3.raw")
```

Comment:
Do you want to make `"application/vnd.github.v3.raw"` a `const` in `github.go`?
No big deal... either way is fine with me.

Reply:
I considered it, but it didn't seem necessary for this initial case. If any other code starts using it then that's probably a good enough time to define it as a constant.